utils/getPositiveNumber.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ *
+ * @param value
+ * @returns
+ */
```

Comment:
I guess we can drop the docs 🙂

Reply:
Or a doc `Returns value if positive or 0 instead.`

Reply:
I will fix the comment